src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -682,14 +685,14 @@ public ICompilationTracker FreezePartialState(CancellationToken cancellationToke
             {
                 var state = this.ReadState();
 
-                var clonedCache = this.SkeletonReferenceCache.Clone();
+                var clonedSkeletonReferenceCache = _skeletonReferenceCache.Clone();
```

Comment:
💡 non-copyable issue should be resolved by inlining this call to clone at the use sites of this local variable